### ISSUE TITLE
App fails to run when deployed to now.sh

### ISSUE THREAD

**Issue by iykekings** [Charca/sapper-blog-template#6]
I cloned and deployed this project as it is to now.sh with [now-sapper](https://github.com/thgh/now-sapper) but I get this when I try to open the app

`502: BAD_GATEWAY Code: NO_STATUS_CODE_FROM_FUNCTION ID: zrh1:5vj8f-1579593325141-fc03a499e23e`

when I check the logs, I found this:

```
Error: Runtime exited with error: exit status 1 Runtime.ExitError
--
Jan 21 2020 08:57:50:435 | 2020-01-21T07:57:50.785Z	undefined	
ERROR	Server is not listening 
Error: ENOENT: no such file or directory, scandir '/var/task/src/routes/blog/posts/'     
at Object.readdirSync (fs.js:854:3)    
 at Object.<anonymous> (/var/task/__sapper__/build/server/server.js:55:18)     
at Module._compile (internal/modules/cjs/loader.js:956:30)     
at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)     
at Module.load (internal/modules/cjs/loader.js:812:32)    
 at Function.Module._load (internal/modules/cjs/loader.js:724:14)     
at Module.require (internal/modules/cjs/loader.js:849:19)     
at require (internal/modules/cjs/helpers.js:74:18)     
at Object.<anonymous> (/var/task/launcher.js:17:14)    
 at Module._compile (internal/modules/cjs/loader.js:956:30) 
{   errno: -2,   syscall: 'scandir',   code: 'ENOENT',   path: '/var/task/src/routes/blog/posts/'
```

**Comment by Charca** [comment]
I haven't tested the `now-sapper` package, but it looks like we might need to adjust the path to the posts folder here: https://github.com/Charca/sapper-blog-template/blob/master/src/routes/blog/_posts.js#L12-L13

You're probably seeing this issue because the build script is not being executed from the root directory of the project. We should fix this in this repo, but if you can find a way to change the directory from where you run the npm script, that should unblock you for now.

**Comment by limitlessloop** [comment]
Hi, I was able to get this working from a fresh copy of sapper and then copying over the components and the blog route files. There was an error with `date-fns` and I have to change the date formatting options and also when I first deployed to Now it complained about there not being an export folder. Changing the date format as the error suggested fixed that issue, and manually exporting the app fixed the Now issue. I need to investigate why Now didn't create the export folder itself though.
